rustls/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "rustls"
 version = "0.21.7"
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.61"
```

Comment:
Should we also bring in the commit that dropped `rust-version` from the internal crates?

Reply:
Sure. Done :ballot_box_with_check: